ui/app/components/app/signature-request/signature-request-message/signature-request-message.component.js

```diff
@@ -0,0 +1,60 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+export default class SignatureRequestMessage extends PureComponent {
+  static propTypes = {
+    data: PropTypes.object.isRequired,
+  }
+
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  makeNode (node) {
+    const leaves = []
+
+    for (const [key, value] of Object.entries(node)) {
+      leaves.push([ key, value ])
+    }
+
+    return leaves
+  }
+
+  renderNode (data) {
+    return (
+      <div className="signature-request-message--node">
+        {this.makeNode(data).map(([ label, value ], i) => (
```

Comment:
From the looks of it, `makeNode` does nothing more than return the result of `Object.entries`, so this could be a call to that method directly.

```suggestion
        {Object.entries(data).map(([ label, value ], i) => (
```

Reply:
And we should be able to remove `makeNode` since it's just the once instance